### ISSUE TITLE
Edit button broken for some site sections

### ISSUE THREAD

**Issue by jeremiahpslewis** [TuringLang/turing.ml#9]
Hey! Thanks for the great package docs. It looks like there is a minor issue with the edit button: not all source files are available in this repo (e.g. the TuringTutorials), but the edit button assumes this repo contains all files, which leads to broken links on pages like this one: https://turing.ml/dev/tutorials/2-logisticregression/

template line at issue is here: https://github.com/TuringLang/turing.ml/blob/77a98d249568ce521af7a97d0e1318f42e0164d3/_includes/sidebar.html#L126

**Comment by jeremiahpslewis** [comment]
![Uploading Screen Shot 2021-03-14 at 18.30.59.png…]()

**Comment by cpfiffer** [comment]
Yeah, we'll revisit this when we launch the new turing.ml internals.
